Content.Server/Construction/Completions/EmptyAllContainers.cs

```diff
@@ -28,9 +36,13 @@ public void PerformAction(EntityUid uid, EntityUid? userUid, IEntityManager enti
             HandsComponent? hands = null;
             var pickup = Pickup && entityManager.TryGetComponent(userUid, out hands);
 
+            EntityCoordinates? dropCoordinates = null;
+            if (EmptyAtUser && entityManager.TryGetComponent(userUid, out TransformComponent? userXform))
+                dropCoordinates = userXform.Coordinates;
+
             foreach (var container in containerManager.GetAllContainers())
             {
-                foreach (var ent in containerSys.EmptyContainer(container, true, reparent: !pickup))
+                foreach (var ent in containerSys.EmptyContainer(container, true, dropCoordinates, reparent: !pickup))
                 {
```

Comment:
Does this work properly if the user is parented? E.g. buckled, as I feel this is just going to actually use the coordinates which isn't valid behavior.